src/i18n/en/strings.txt

```diff
@@ -81,3 +82,8 @@ description=Description
 edit=Edit
 delete=Delete
 offline=Offline
+site=Site
+change=Change
+save=Save
+default_journal=Default journal
+settings=settings
```

Comment:
hm I guess I could easily do it with js or css in code.... but it's been used as is through the entire app...